corehq/apps/users/util.py

```diff
@@ -126,18 +126,12 @@ def user_id_to_username(user_id, use_name_if_available=False):
         return raw_username(user_object['username']) if "username" in user_object else None
 
 
+@quickcache(['user_id'])
```

Comment:
I think adding this is going to have the same effect as completely clearing the cache for this function after it is deployed, right? Are there performance implications for that?

Reply:
The default timeout setting for `cache.set` below is 5 minutes (we don't set it in localsettings.py.j2 and 5 min is the Django default https://docs.djangoproject.com/en/2.2/ref/settings/#std:setting-CACHES-TIMEOUT), so it's already only helpful for multiple calls in the same workload or burst, not for saving it from one day to the next or anything. So my estimate based on that is that caching is important, but starting from a warm cache isn't.

Thanks for asking that question. It's a great one that had slipped my mind.